app/assets/stylesheets/ltags/InstagramTag.scss

```diff
@@ -2,15 +2,18 @@
   background: rgb(255, 255, 255);
   border: 1px solid rgb(219, 219, 219);
   margin: 1px 1px 12px;
-  max-width: 450px;
+  min-width: 326px;
```

Comment:
These CSS changes come from my research of the embed codes that Instagram generates. I updated some values to match Instagram's embed codes.